source/MyProcessing.mc

```diff
@@ -355,6 +352,9 @@ class MyProcessing {
       if(self.iAccuracy > Pos.QUALITY_LAST_KNOWN) {
         self.iPositionEpoch = _iEpoch;
 
+        // ... wind
+        self.windStep();
```

Comment:
My reasoning for moving this is I think we should always use stateful position for wind? LMK if this is wrong

Reply:
Yeah I think that's fine!